terraform/modules/services/api_pvsite/eb.tf

```diff
@@ -61,6 +61,24 @@ resource "aws_elastic_beanstalk_environment" "eb-api-env" {
     value     = var.sentry_dsn
   }
 
+  setting {
```

Comment:
One other thing, need to add it to this too - https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/modules/services/api_pvsite/docker-compose.yml